src/components/Table/TableRow/TableRow.types.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export type TableRowProps = {
+    /**
+     * Any additional class names to apply
+     */
+    className?: React.ReactNode;
+
+    /**
+     * What type of spacing the cell should have
+     */
+    height?: 'regular' | 'relaxed';
```

Comment:
I'm not convinced with `regular` and `relaxed` but medium/big ... is not better. So I don't have a better idea.

Reply:
In Gmail, they have the naming convention, `default`, `comfortable`, and `compact`. We can think about it some more. It doesn't have to be finalized until we need to make the release.

Reply:
In Gmail, it changes the padding top/bottom to change the density of each row whereas for us we just change the height because we want to display two lines in a cell but we keep the same density. 
Maybe `height?: oneLine | twoLine`. But it's very descriptive.
We can keep it like this for now, and discuss it during our next meeting.

Reply:
After a discussion with Samson, as we want to avoid people adding the wrong height value for the single-line row. We think it can be clearer to replace the height property by: `isMultipleLine ?: boolean`. What do you think?

Reply:
That sounds good to me.